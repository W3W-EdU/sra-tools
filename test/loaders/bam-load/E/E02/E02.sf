#bam-load error test #02 : an aligment has an optional 'OQ'-tag ( orig. quality )
#but of different lenght then SEQUENCE

r:type=random,name=R1,length=5000

# write all references into one file as FASTA
ref-out:E02.fasta

# write the final SAM-output into this file
sam-out:E02.sam

# write a config-file for bam-load
config:E02.config

p:name=A1,pos=100,cigar=30M,opts=OQ:Z:actfgrh0123
p:name=A1,pos=300

#some additional ( correct ) alignments
p:name=B,repeat=50
p:name=B,repeat=50
