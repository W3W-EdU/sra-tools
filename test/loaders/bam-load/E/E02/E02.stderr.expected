err: tools/loaders/bam-loader/bam.c:2711:BAM_AlignmentGetQuality2: data inconsistent while reading document within alignment module - Spot 'A1': length of original quality does not match sequence
err: libs/kdb/wkdb.c:1193:KDBVDrop: data inconsistent while reading document within alignment module - load failed
